src/Servers/IIS/IIS/src/Core/OutputProducer.cs

```diff
@@ -51,7 +50,7 @@ public void Complete()
         }
     }
 
-    public void Abort(Exception error)
```

Comment:
Should the pipe be completed with this exception?

Reply:
```diff
- _pipe.Writer.Complete();
+ _pipe.Writer.Complete(error);
```

Like so?

Reply:
Yeah, would have to look at the usage to see if we want that though.